account_invoice_inter_company/__manifest__.py

```diff
@@ -15,6 +15,8 @@
     'depends': [
         'account_accountant',
         'onchange_helper',
+        'partner_multi_company',
```

Comment:
These dependencies shouldn't be needed, as it's adding an unneeded overhead.

Reply:
See upper versions for differences, as we faced similar problems and we solved them

Reply:
That is done with knowledge. As it is highlighted there : https://github.com/OCA/multi-company/blob/10.0/account_invoice_inter_company/tests/test_inter_company_invoice.py#L52

It seems logical to manage multi company on partners and products in that flow.

As the modifications done in upper branches are too difficult to backport as is, I think this solution is not so bad.

I did this in order to unlock some people, taking the time to test every change I've done. If the solution do not fit to your requirements, let come others who will finish the work, I stop here

Reply:
Well, for me that sounds as a bad patch someone did in the tests to fix their flow, but the module doesn't require that modules for sure. Anyway, I'm not using anymore 10.0, so for me is the same. It was just for not adding suddenly 2 modules as dependencies for those that have this module installed, so I abstain from approving nor rejecting this PR, and let others to decide.